src/leaflike/bookmarks.clj

```diff
@@ -197,3 +198,39 @@
                                                                     :all-tags all-tags))
                                         :error-msg error-msg))
         (assoc-in [:headers "Content-Type"] "text/html"))))
+
+(defn pocket-import-form
+  [{:keys [params] :as request}]
+  (-> (res/response (layout/user-view "Import from pocket"
+                                      (get-in request [:session :username])
+                                      (views/pocket-import-form anti-forgery/*anti-forgery-token*)))
+      (assoc-in [:headers "Content-Type"] "text/html")))
+
+(let [get-url (fn [node]
```

Comment:
Rather than nesting `import-bookmarks-from-pocket` inside the `let` so that it has access to all the `get-` fns, you can move the `get-` fns inside `import-bookmarks-from-pocket` if the intention is not have separate top-level fns for these.

Reply:
@trycatcher We're defining these fn's outside `import-bookmarks-from-pocket` because we don't want the functions to get re-defined on each call. They don't use any of the args for the `import-bookmarks-from-pocket` fn but we don't want the `get-*` fn's to be top-level either.